test/integration/inject/inject_test.go

```diff
@@ -299,6 +302,49 @@ func TestInjectAutoPod(t *testing.T) {
 		k8s.ProxyInjectAnnotation: k8s.ProxyInjectEnabled,
 	}
 
+	truthy := true
+	falsy := false
+	zero := int64(0)
+	expectedInitContainer := v1.Container{
+		Name:  k8s.InitContainerName,
+		Image: "gcr.io/linkerd-io/proxy-init:" + version.ProxyInitVersion,
+		Args: []string{
+			"--incoming-proxy-port", "4143",
+			"--outgoing-proxy-port", "4140",
+			"--proxy-uid", "2102",
+			"--inbound-ports-to-ignore", "4190,4191,1234,5678",
```

Comment:
We should only expect `4190` and `4191` right?

Reply:
I'm testing the `global.proxyInit.ignoreInboundPorts=1234,5678` override I added to `install_test.go`. I'll add a comment.